core/kt-osrd-path/src/main/kotlin/fr/sncf/osrd/path/implementations/TrainPathBuilder.kt

```diff
@@ -253,7 +255,7 @@ internal fun generateRouteRanges(
     routes: List<RouteId>,
 ): LinearObjectMap<RouteRange> {
     val res = mutableListOf<RouteRange>()
-    val mappedChunks = chunks.asMapOfRanges().map { it.value }.associateBy { it.value }
+    val mappedChunks = chunks.entries.map { it.value }.associateBy { it.value }
```

Comment:
It looks like you can do:
```suggestion
    val mappedChunks = chunks.values.associateBy { it.value }
```